arcade/scene.py

```diff
@@ -240,6 +240,13 @@ def remove_sprite_list_by_name(
         del self._name_mapping[name]
 
     def remove_sprite_list_by_object(self, sprite_list: SpriteList) -> None:
+        """
+        Remove a SpriteList from the Scene.
+
+        This function serves to completely remove the SpriteList from the Scene.
+
+        :param str SpriteList: The SpriteList to remove.
```

Comment:
The typing/name for the `sprite_list` parameter here need corrected. The type should be `SpriteList` and the parameter name should be `sprite_list`

Reply:
sry, lol

Reply:
I changed the wrong thing.